apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -107,11 +108,12 @@
      * Check that selected element is fully visible on the screen
      */
     isFullyVisible: function (element) {
-      var height = document.documentElement.clientHeight;
-      var elementRects = element.getClientRects()[0];
-      if (elementRects.bottom > height) {
+      var gaiaGrid = document.getElementsByTagName('gaia-grid-rs')[0];
+      var gridRects = gaiaGrid.getBoundingClientRect();
```

Comment:
Why not to rename both these variables ``gridRects`` && ``elementRects`` to singular? If I understand correctly now you are assigning single rectangle to each of them.